mmv1/products/securitycenterv2/product.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: SecurityCenterV2
+display_name: Security Command Center (SCC)
```

Comment:
Could you modify this to indicate that it is the v2 version? For other products, we've added `(v2 API)` to the end

Reply:
It looks like this wasn't addressed either?

Reply:
addressed